runtime/amplitude/src/lib.rs

```diff
@@ -275,7 +256,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("amplitude"),
 	impl_name: create_runtime_str!("amplitude"),
 	authoring_version: 1,
-	spec_version: 17,
+	spec_version: 18,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 13,
```

Comment:
I thought about it, I deliberately left it unchanged but we have to discuss it, since changing it will brake integrations, but not changing it may also cause confusion if the calls have changed substantially.

Reply:
I think it should be fine if we continue to leave it as is. So far, I'm only aware of the Zondax (ledger) team relying on it so I would assume that the other wallets don't necessarily check that number.